app/controllers/people_controller.rb

```diff
@@ -9,6 +9,9 @@ def show
     data = Parliament::Request.new.people(person_id).get
 
     @person = data.filter('http://id.ukpds.org/schema/Person').first
+
+    @current_seat_incumbency = @person.seat_incumbencies.select(&:current?).first
+    @current_party_membership = @person.party_memberships.select(&:current?).first
```

Comment:
We need to add some tests for this, I'll make sure it's addressed in the upcoming sprint